### PR TITLE
Add release notes for Kyma 2.15

### PR DESCRIPTION
**Pull request by Sawthis**
**Description**

Changes proposed in this pull request:

- Add release notes for Kyma 2.15

**Related issue(s)**

### CODE REVIEW

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+TBD
+
+## General
+For upgrading to Kyma 2.15 please read the migration guide
+
+## Telemetry
+### Manager
+- In preparation ((https://github.com/kyma-project/telemetry-manager/issues/150)) for turning the telemetry component into a module, resources got consolidated and a cleanup script need to be executed on upgrade. Please follow the related migration script
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:8]:
```suggestion
- In preparation ((https://github.com/kyma-project/telemetry-manager/issues/150)) for turning the Telemetry component into a module, resources have been consolidated. On upgrade, you must execute a cleanup script. Please follow the [migration script](LINK-TO-SCRIPT).
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- More robust handling of webhook certificates (https://github.com/kyma-project/kyma/issues/16626)
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:9]:
```suggestion
- Handling of webhook certificates (https://github.com/kyma-project/kyma/issues/16626) has been improved.
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+### Tracing
+- Update of components
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:12]:
```suggestion
- Updated OTel Collector to 0.77.0 (https://github.com/kyma-project/kyma/pull/17469).
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  - Otel Collector 0.77.0 (https://github.com/kyma-project/kyma/pull/17469)
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:13]:
```suggestion
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+### Logging
+- Update of components
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:16]:
```suggestion
- Updated Fluent Bit to 2.1.2 (https://github.com/kyma-project/kyma/pull/17485).
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+ - Fluent Bit 2.1.2 (https://github.com/kyma-project/kyma/pull/17485)
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:17]:
```suggestion
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+- Improved security setup of fluentbit (https://github.com/kyma-project/kyma/pull/17574)
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:18]:
```suggestion
- Improved security setup of Fluent Bit (https://github.com/kyma-project/kyma/pull/17574).
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 8]
```diff
@@ -1,11 +1,8 @@
-TBD
-
-## General
-For upgrading to Kyma 2.15 please read the migration guide
+Still warm, as the June sun at noon, comes Kyma 2.15. The hot season has already started, so juicy improvements are here to refresh your experience with our product. Sip the strawberry-Telementry, watermelon-Security, raspberry-API Gateway, and peach-Serverless cool punch news as if you were on a Hawaiian beach. Read on to see what we prepared for you!
 
 ## Telemetry
 ### Manager
-- In preparation ((https://github.com/kyma-project/telemetry-manager/issues/150)) for turning the telemetry component into a module, resources got consolidated and a cleanup script need to be executed on upgrade. Please follow the related migration script
+- In [preparation for turning the telemetry component into a module](https://github.com/kyma-project/telemetry-manager/issues/150), resources got consolidated. As a result, you need to run a cleanup script when you upgrade to Kyma version 2.15. For more details, read the [2.14-2.15 Migration Guide](https://github.com/kyma-project/kyma/blob/main/docs/migration-guide-2.14-2.15.md).
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:5]:
```suggestion
- In [preparation for turning the telemetry component into a module](https://github.com/kyma-project/telemetry-manager/issues/150), resources have been consolidated. As a result, you must run a cleanup script when you upgrade to Kyma version 2.15. For more details, read the [2.14-2.15 Migration Guide](https://github.com/kyma-project/kyma/blob/main/docs/migration-guide-2.14-2.15.md).
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
+Still warm, as the June sun at noon, comes Kyma 2.15. The hot season has already started, so juicy improvements are here to refresh your experience with our product. Sip the strawberry-Telementry, watermelon-Security, raspberry-API Gateway, and peach-Serverless cool punch news as if you were on a Hawaiian beach. Read on to see what we prepared for you!
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:1]:
```suggestion
Still warm, as the June sun at noon, comes Kyma 2.15. The hot season has already started, so juicy improvements are here to refresh your experience with our product. Sip the strawberry-Telemetry, watermelon-Security, raspberry-API Gateway, and peach-Serverless cool punch news as if you were on a Hawaiian beach. Read on to see what we prepared for you!
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+## Telemetry
+### Manager
+- In [preparation for turning the telemetry component into a module](https://github.com/kyma-project/telemetry-manager/issues/150), resources have been consolidated. As a result, you must run a cleanup script when you upgrade to Kyma version 2.15. For more details, read the [2.14-2.15 Migration Guide](https://github.com/kyma-project/kyma/blob/main/docs/migration-guide-2.14-2.15.md).
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:5]:
```suggestion
- In [preparation for turning the Telemetry component into a module](https://github.com/kyma-project/telemetry-manager/issues/150), resources have been consolidated. As a result, you must run a cleanup script when you upgrade to Kyma version 2.15. For more details, read the [2.14-2.15 Migration Guide](https://github.com/kyma-project/kyma/blob/main/docs/migration-guide-2.14-2.15.md).
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+## API Gateway
+### Default timeout for HTTP requests
+This Kyma release brings unified timeout for workloads exposed with APIRules. The default timeout for HTTP requests is now 180s and it’s defined on the Istio VirtualService level.
+
+### Ory Hydra Oauth2Client migration 
+The Ory stack has been deprecated since Kyma version 2.2. The recently published [blog post](https://blogs.sap.com/2023/06/06/sap-btp-kyma-runtime-ory-hydra-oauth2-client-migration/) explains how to migrate from the Ory Hydra client to the application created in SAP Cloud Identity Services. Start the migration process as soon as possible. 
+
+## Serverless
+### Simplified internal Docker registry setup
+With Kyma 2.15 we simplified Serverless configuration for the internal Docker registry. From now on, the images for Function runtime Pods are pulled from the internal Docker registry via NodePort.
```
Comment (NHingerl) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:27]:
```suggestion
With Kyma 2.15 we simplified Serverless configuration for the internal Docker registry. From now on, the images for Function runtime Pods are pulled from the internal Docker registry with NodePort.
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
+Still warm, as the June sun at noon, comes Kyma 2.15. The hot season has already started, so juicy improvements are here to refresh your experience with our product. Sip the strawberry-Telemetry, watermelon-Security, raspberry-API Gateway, and peach-Serverless cool punch news as if you were on a Hawaiian beach. Read on to see what we prepared for you!
+
+## Telemetry
+### Manager
+- In [preparation for turning the Telemetry component into a module](https://github.com/kyma-project/telemetry-manager/issues/150), resources have been consolidated. As a result, you must run a cleanup script when you upgrade to Kyma version 2.15. For more details, read the [2.14-2.15 Migration Guide](https://github.com/kyma-project/kyma/blob/main/docs/migration-guide-2.14-2.15.md).
+- Handling of webhook certificates (https://github.com/kyma-project/kyma/issues/16626) has been improved.
+
+### Tracing
+- Updated OTel Collector to 0.77.0 (https://github.com/kyma-project/kyma/pull/17469).
```
Comment (IwonaLanger) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:9]: Wouldn't it look nicer if you used inline links in this section?

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+The Ory stack has been deprecated since Kyma version 2.2. The recently published [blog post](https://blogs.sap.com/2023/06/06/sap-btp-kyma-runtime-ory-hydra-oauth2-client-migration/) explains how to migrate from the Ory Hydra client to the application created in SAP Cloud Identity Services. Start the migration process as soon as possible. 
+
+## Serverless
+### Simplified internal Docker registry setup
+With Kyma 2.15 we simplified Serverless configuration for the internal Docker registry. From now on, the images for Function runtime Pods are pulled from the internal Docker registry with NodePort.
+
+With this change, we improve security as the internal Docker registry is no longer exposed outside of the Kubernetes cluster. Additionally, it makes Serverless fully independent from the Istio module in all installation modes.
+
+### Deployment profiles removed
+In preparation for an independent installation model, we removed the predefined deployment profiles, namely evaluation, and production, for Serverless. We are shifting from profiled overrides used during module installation towards runtime-configurable resources.
```
Comment (IwonaLanger) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:32]:
```suggestion
In preparation for an independent installation model, we removed the predefined deployment profiles for Serverless, namely evaluation, and production. We are shifting from profiled overrides used during module installation towards runtime-configurable resources.
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 14]
```diff
@@ -0,0 +1,32 @@
+Still warm, as the June sun at noon, comes Kyma 2.15. The hot season has already started, so juicy improvements are here to refresh your experience with our product. Sip the strawberry-Telemetry, watermelon-Security, raspberry-API Gateway, and peach-Serverless cool punch news as if you were on a Hawaiian beach. Read on to see what we prepared for you!
+
+## Telemetry
+### Manager
+- In [preparation for turning the Telemetry component into a module](https://github.com/kyma-project/telemetry-manager/issues/150), resources have been consolidated. As a result, you must run a cleanup script when you upgrade to Kyma version 2.15. For more details, read the [2.14-2.15 Migration Guide](https://github.com/kyma-project/kyma/blob/main/docs/migration-guide-2.14-2.15.md).
```
Comment (grego952) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:5]:
```suggestion
- In [preparation for turning the Telemetry component into a module](https://github.com/kyma-project/telemetry-manager/issues/150), resources have been consolidated. As a result, you must run a cleanup script when you upgrade to Kyma version 2.15. For more details, read the [2.14-2.15 Migration Guide](https://github.com/kyma-project/kyma/blob/release-2.15/docs/migration-guide-2.14-2.15.md).
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 15]
```diff
@@ -0,0 +1,32 @@
...
+- Handling of webhook certificates (https://github.com/kyma-project/kyma/issues/16626) has been improved.
```
Comment (grego952) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:6]:
```suggestion
- [Handling of webhook certificates](https://github.com/kyma-project/kyma/issues/16626) has been improved.
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 16]
```diff
@@ -0,0 +1,32 @@
...
+
+### Tracing
+- Updated OTel Collector to 0.77.0 (https://github.com/kyma-project/kyma/pull/17469).
+
+### Logging
+- Updated Fluent Bit to 2.1.2 (https://github.com/kyma-project/kyma/pull/17485).
+- Improved security setup of Fluent Bit (https://github.com/kyma-project/kyma/pull/17574).
+
+## Security
+With the 2.15 Kyma version, `ECDHE-RSA-AES256-SHA` and `ECDHE-RSA-AES128-SHA` cipher suites used in the default Kyma gateway become deprecated as part of security hardening and following Kyma security team recommendations. The configurations will be removed with Kyma version 2.18 and clients dependent on the cipher suites won’t be accepted.
```
Comment (grego952) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:16]:
```suggestion
With the 2.15 Kyma version, `ECDHE-RSA-AES256-SHA` and `ECDHE-RSA-AES128-SHA` cipher suites used in the default Kyma gateway become deprecated as part of security hardening and following Kyma security team recommendations. The configurations will be removed with Kyma version 2.18, and clients dependent on the cipher suites won’t be accepted.
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 17]
```diff
@@ -0,0 +1,32 @@
...
+The Ory stack has been deprecated since Kyma version 2.2. The recently published [blog post](https://blogs.sap.com/2023/06/06/sap-btp-kyma-runtime-ory-hydra-oauth2-client-migration/) explains how to migrate from the Ory Hydra client to the application created in SAP Cloud Identity Services. Start the migration process as soon as possible. 
+
+## Serverless
+### Simplified internal Docker registry setup
+With Kyma 2.15 we simplified Serverless configuration for the internal Docker registry. From now on, the images for Function runtime Pods are pulled from the internal Docker registry with NodePort.
+
+With this change, we improve security as the internal Docker registry is no longer exposed outside of the Kubernetes cluster. Additionally, it makes Serverless fully independent from the Istio module in all installation modes.
+
+### Deployment profiles removed
+In preparation for an independent installation model, we removed the predefined deployment profiles, namely evaluation, and production for Serverless. We are shifting from profiled overrides used during module installation towards runtime-configurable resources.
```
Comment (grego952) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:32]:
```suggestion
In preparation for an independent installation model, we removed the predefined deployment profiles, namely evaluation and production for Serverless. We are shifting from profiled overrides used during module installation towards runtime-configurable resources.
```
I think we don't need this comma here.

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 18]
```diff
@@ -0,0 +1,32 @@
...
+### Logging
+- Updated Fluent Bit to 2.1.2 (https://github.com/kyma-project/kyma/pull/17485).
+- Improved security setup of Fluent Bit (https://github.com/kyma-project/kyma/pull/17574).
+
+## Security
+With the 2.15 Kyma version, `ECDHE-RSA-AES256-SHA` and `ECDHE-RSA-AES128-SHA` cipher suites used in the default Kyma gateway become deprecated as part of security hardening and following Kyma security team recommendations. The configurations will be removed with Kyma version 2.18 and clients dependent on the cipher suites won’t be accepted.
+
+## API Gateway
+### Default timeout for HTTP requests
+This Kyma release brings unified timeout for workloads exposed with APIRules. The default timeout for HTTP requests is now 180s and it’s defined on the Istio VirtualService level.
```
Comment (nataliasitko) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:20]:
```suggestion
This Kyma release brings a unified timeout for workloads exposed with APIRules. The default timeout for HTTP requests is now 180s, and it’s defined on the Istio VirtualService level.
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 19]
```diff
@@ -0,0 +1,32 @@
...
+
+### Ory Hydra Oauth2Client migration
```
Comment (nataliasitko) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:22]:
```suggestion
### Ory Hydra OAuth2 client migration 
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 20]
```diff
@@ -0,0 +1,32 @@
...
+## API Gateway
+### Default timeout for HTTP requests
+This Kyma release brings unified timeout for workloads exposed with APIRules. The default timeout for HTTP requests is now 180s and it’s defined on the Istio VirtualService level.
+
+### Ory Hydra Oauth2Client migration 
+The Ory stack has been deprecated since Kyma version 2.2. The recently published [blog post](https://blogs.sap.com/2023/06/06/sap-btp-kyma-runtime-ory-hydra-oauth2-client-migration/) explains how to migrate from the Ory Hydra client to the application created in SAP Cloud Identity Services. Start the migration process as soon as possible. 
+
+## Serverless
+### Simplified internal Docker registry setup
+With Kyma 2.15 we simplified Serverless configuration for the internal Docker registry. From now on, the images for Function runtime Pods are pulled from the internal Docker registry with NodePort.
```
Comment (nataliasitko) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:27]:
```suggestion
With Kyma 2.15, we simplified Serverless configuration for the internal Docker registry. From now on, the images for Function runtime Pods are pulled from the internal Docker registry with NodePort.
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md (hunk 12 above)
Comment (mmitoraj) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:9]:
```suggestion
- [Updated OTel Collector to 0.77.0](https://github.com/kyma-project/kyma/pull/17469).
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 21]
```diff
@@ -0,0 +1,32 @@
...
+## Telemetry
+### Manager
+- In [preparation for turning the Telemetry component into a module](https://github.com/kyma-project/telemetry-manager/issues/150), resources have been consolidated. As a result, you must run a cleanup script when you upgrade to Kyma version 2.15. For more details, read the [2.14-2.15 Migration Guide](https://github.com/kyma-project/kyma/blob/main/docs/migration-guide-2.14-2.15.md).
+- Handling of webhook certificates (https://github.com/kyma-project/kyma/issues/16626) has been improved.
+
+### Tracing
+- Updated OTel Collector to 0.77.0 (https://github.com/kyma-project/kyma/pull/17469).
+
+### Logging
+- Updated Fluent Bit to 2.1.2 (https://github.com/kyma-project/kyma/pull/17485).
```
Comment (mmitoraj) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:12]:
```suggestion
- [Updated Fluent Bit to 2.1.2](https://github.com/kyma-project/kyma/pull/17485).
```

File: docs/kyma-release-notes/2023-06-07-release-notes-2.15.md [hunk 22]
```diff
@@ -0,0 +1,32 @@
...
+- Improved security setup of Fluent Bit (https://github.com/kyma-project/kyma/pull/17574).
```
Comment (mmitoraj) [inline docs/kyma-release-notes/2023-06-07-release-notes-2.15.md:13]:
```suggestion
- [Improved security setup of Fluent Bit](https://github.com/kyma-project/kyma/pull/17574).
```

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**